### PR TITLE
trim kernel config down for unnecessary features

### PR DESCRIPTION
**Pull request by dmacattack**
* remove unnecessary drivers and features

### CODE REVIEW

File: board/inspectron/wiscopeMurata/wiscope_linux_defconfig [hunk 1]
```diff
@@ -1067,67 +1070,15 @@ CONFIG_NET_FLOW_LIMIT=y
 # end of Networking options
 
 # CONFIG_HAMRADIO is not set
-CONFIG_CAN=y
```
Comment (dmacattack) [inline board/inspectron/wiscopeMurata/wiscope_linux_defconfig:1070]: dont need CAN - hummingboard supports it, but you're not using it

File: board/inspectron/wiscopeMurata/wiscope_linux_defconfig [hunk 2]
```diff
@@ -1067,67 +1070,15 @@ CONFIG_NET_FLOW_LIMIT=y
...
-
+# CONFIG_CAN is not set
 CONFIG_BT=y
 CONFIG_BT_BREDR=y
-CONFIG_BT_RFCOMM=y
-# CONFIG_BT_RFCOMM_TTY is not set
-# CONFIG_BT_BNEP is not set
-CONFIG_BT_HIDP=y
-CONFIG_BT_HS=y
+# CONFIG_BT_RFCOMM is not set
```
Comment (dmacattack) [inline board/inspectron/wiscopeMurata/wiscope_linux_defconfig:1076]: you can add these bluetooth features later as needed, but i was trying to slim it down to the essentials

File: board/inspectron/wiscopeMurata/wiscope_linux_defconfig [hunk 3]
```diff
@@ -1139,36 +1090,28 @@ CONFIG_BT_DEBUGFS=y
...
-CONFIG_BT_INTEL=y
-CONFIG_BT_BCM=y
-CONFIG_BT_RTL=y
-CONFIG_BT_HCIBTUSB=y
-# CONFIG_BT_HCIBTUSB_AUTOSUSPEND is not set
-CONFIG_BT_HCIBTUSB_BCM=y
-# CONFIG_BT_HCIBTUSB_MTK is not set
-CONFIG_BT_HCIBTUSB_RTL=y
+CONFIG_BT_BCM=m
+# CONFIG_BT_HCIBTUSB is not set
```
Comment (dmacattack) [inline board/inspectron/wiscopeMurata/wiscope_linux_defconfig:1094]: i disabled the bluetooth USB features - your bluetooth isnt a usb dongle

File: board/inspectron/wiscopeMurata/wiscope_linux_defconfig [hunk 4]
```diff
@@ -1139,36 +1090,28 @@ CONFIG_BT_DEBUGFS=y
...
 # CONFIG_BT_HCIBTSDIO is not set
-CONFIG_BT_HCIUART=y
+CONFIG_BT_HCIUART=m
```
Comment (dmacattack) [inline board/inspectron/wiscopeMurata/wiscope_linux_defconfig:1096]: made it loadable for testing - you can put it back when you figure it all out

**Review by gauthamkantharaju — APPROVED** [review]

**Pull request merged**